benchmark_utils/transformation.py

```diff
@@ -6,15 +6,14 @@
 # - getting requirements info when all dependencies are not installed.
 with safe_import_context() as import_ctx:
     import numpy as np
-
     from numpy import concatenate
```

Comment:
prefer using `np.concatenate` instead of importing the function, as this makes it easier to read the code.